Source/CarthageKit/Xcode.swift

```diff
@@ -508,10 +509,23 @@ public struct BuildSettings {
 	}
 
 	/// Attempts to determine the SDK this scheme builds for.
-	public var buildSDK: Result<SDK, CarthageError> {
+	public var firstBuildSDK: Result<SDK, CarthageError> {
 		return self["PLATFORM_NAME"].flatMap(SDK.fromString)
 	}
 
+	public var buildSDK: SignalProducer<SDK, CarthageError> {
```

Comment:
This should probably be renamed to reflect that it can send multiple SDKs. Also, it could use some documentation.